content/docs/guides/pipelines/build-component.md

```diff
@@ -172,3 +172,6 @@ See [an example](https://github.com/kubeflow/pipelines/blob/master/samples/xgboo
 You can also build lightweight components from Python functions. See the guide 
 to 
 [lightweight python components](/docs/guides/pipelines/lightweight-python-components).
+
+## Export metrics
```

Comment:
For code readability, please add an empty line after the heading.

Reply:
done